README.md

```diff
@@ -51,9 +57,9 @@ identity messages **MUST** conform to the following rules:
 
  - Has a `type` field mapping to a BFE string (i.e. `<06 00> + data`)
  which can assume only one the following possible values:
-   - `networkidentity/add`
-   - `networkidentity/tombstone`
- - Has a `networkidentity` field mapping to a BFE "network identity
+   - `network-identity/add`
+   - `network-identity/tombstone`
+ - Has a `network-identity` field mapping to a BFE "network identity
    ID", i.e. `<07 01> + data`
  - Has a `metafeed` field mapping to a BFE "Bendy Butt feed ID", i.e.
  `<00 03> + data`
```

Comment:
The paragraph below mentions "MUST use the `identity`'s cryptographic keypair". What `identity` is this? It's not mentioned elsewhere

Reply:
Right, for derived identities we should include nonce. WiIl fix

Reply:
What is `identity`?

Reply:
Added another commit to https://github.com/ssb-ngi-pointer/ssb-network-identity-spec/pull/2 to try and improve this part.